src/test/java/bio/terra/app/controller/SnapshotsApiControllerTest.java

```diff
@@ -163,7 +164,7 @@ private static Stream<Arguments> testQuerySnapshotData() {
                 .content(
                     TestUtils.mapToJson(
                         new QueryDataRequestModel()
-                            .direction(DIRECTION)
+                            .direction(SqlSortDirectionAscDefault.ASC)
```

Comment:
Either this, or you could change `DIRECTION` to be `SqlSortDirectionAscDefault.ASC`  and use 
```
            SqlSortDirection.from(DIRECTION),
```

below, to keep the code using the same constant everywhere.